src/interpreter/mod.rs

```diff
@@ -0,0 +1,81 @@
+use std::collections::HashMap;
+use bytecode::LuaBytecode;
+use bytecode::instructions::Value;
+use bytecode::instructions::Instr::*;
+
+/// Represents a `LuaBytecode` interpreter.
+pub struct Interpreter<'a> {
+    bytecode: LuaBytecode,
+    env: HashMap<&'a String, Value>
```

Comment:
Ultimately I suspect we want this to be something like `frames: Vec<Frame>` and then each `Frame` to have a `Vec` of registers (or similar), all indexed as integers. But that's for a future PR.

Reply:
As I said in the comment above, at the moment I assume I only have a block, but when I introduce functions I will need something like this (or even before that).